backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserAuthorizationVerifier.java

```diff
@@ -99,11 +101,16 @@ public boolean userHasPermission(UserPermission permission) {
 
   public boolean userIsInSameOrg(UUID userId) {
     Optional<OrganizationRoles> currentOrgRoles = _orgService.getCurrentOrganizationRoles();
-    String otherUserEmail = getUser(userId).getLoginEmail();
+    ApiUser otherUser = getUser(userId);
+    String otherUserEmail = otherUser.getLoginEmail();
     Optional<Organization> otherOrg =
         _oktaRepo
             .getOrganizationRoleClaimsForUser(otherUserEmail)
             .map(r -> _orgService.getOrganization(r.getOrganizationExternalId()));
+    if (_featureFlagsConfig.isOktaMigrationEnabled()) {
+      otherOrg = otherUser.getOrganizations().stream().findFirst();
```

Comment:
⚠️ This assumes that the user will have one org associated with them since it's not possible to be part of more than one org. Let me know if you think this should be more robust.